src/vs/workbench/contrib/notebook/browser/controller/editActions.ts

```diff
@@ -103,6 +103,12 @@ registerAction2(class QuitEditCellAction extends NotebookCellAction {
 						primary: KeyCode.Escape,
 						weight: NOTEBOOK_EDITOR_WIDGET_ACTION_WEIGHT - 5
 					},
+					{
+						when: ContextKeyExpr.and(NOTEBOOK_EDITOR_FOCUSED,
```

Comment:
@rebornix - you had a concern about needing to escape from suggestion boxes, but are those even used in webviews?